lib/backend/providers.dart

```diff
@@ -42,7 +45,15 @@ class _ProvidersSingleton {
 
   Future<void> _init() async {
     if (_inited) return;
-    _isar = await openIsar();
+    WidgetsFlutterBinding.ensureInitialized();
+    final dir = await getApplicationDocumentsDirectory();
```

Comment:
was the old database inited in the same dir? with the same name? i would like to keep retro compatibility

Reply:
I see openIsar now doesnt exist in v1.0, hence we need to change its initialisation way.
In docs too author mentioned 
```Breaking 
The internal database format has been changed to improve performance. Old databases do not work anymore!
```
hence backward compatibility might not be possible.

Going forward, we need to check if we are defining path of isar correctly as per we have decided or not.